context/package.json

```diff
@@ -62,6 +62,7 @@
     "dev-server": "webpack-dev-server --config build-utils/webpack.dev.js --hot",
     "test": "jest",
     "test:watch": "jest --watch",
-    "test:coverage": "jest --coverage"
+    "test:coverage": "jest --coverage",
+    "lint-fix": "eslint -c .eslintrc.yml --ext .js --fix app/static/js/"
```

Comment:
- I think we're also using `.jsx` extensions?
- Mention in the readme?

Reply:
Oh right, I have updated this.